Pcap++/src/DpdkDeviceList.cpp

```diff
@@ -258,24 +258,25 @@ bool DpdkDeviceList::verifyHugePagesAndDpdkDriver()
 	execResult = executeShellCommand("lsmod | grep -s igb_uio");
 	if (execResult == "")
 	{
-		execResult = executeShellCommand("modinfo -d uio_pci_generic");
-		if (execResult.find("ERROR") != std::string::npos)
+		try
 		{
-			execResult = executeShellCommand("modinfo -d vfio-pci");
-			if (execResult.find("ERROR") != std::string::npos)
+			execResult = executeShellCommand("modinfo -d uio_pci_generic");
+			PCPP_LOG_DEBUG("uio_pci_generic module is loaded");
+		}
+		catch (const std::runtime_error&)
+		{
+			try
 			{
-				PCPP_LOG_ERROR("None of igb_uio, uio_pci_generic, vfio-pci kernel modules are loaded so DPDK cannot be initialized. Please run <PcapPlusPlus_Root>/setup_dpdk.sh");
-				return false;
+				execResult = executeShellCommand("modinfo -d vfio-pci");
+				PCPP_LOG_DEBUG("vfio-pci module is loaded");
 			}
-			else
+			catch (const std::runtime_error&)
```

Comment:
why not print the error?

Reply:
Imo, the original error message is sufficiently descriptive if the entire try-catch chain fails.

Also printing in the last catch will only ever print the last (vfio-pci) command as error.

We can print it as a debug message, but we already have debug messages on success so idk.